verified-reviews/utils/client.ts

```diff
@@ -26,8 +27,8 @@ const orderMap = {
   date_ASC: "date_asc",
   rate_DESC: "rate_desc",
   rate_ASC: "rate_asc",
-  helpfulrating_DESC: "most_helpful",
-} as const;
+  helpfulrating_DESC: "helpfulrating_desc",
```

Comment:
```suggestion
  helpfulrating_DESC: "most_helpful" as const,
```

keep it as was before. It may be working for some current client. If needed, the client will customize.